### ISSUE TITLE
nested_select doesn't work on deeply nested form 

### ISSUE THREAD

**Issue by keyurvi** [platanus/activeadmin_addons#283]
```ruby
form do |f|
  f.semantic_errors *f.object.errors.keys
  panel object.user_name do 
    f.inputs "Hindustani" do
      f.input :user_name
    end
    f.inputs "child" do
      f.has_many :children, new_record: "Add a child", allow_destroy: true do |ff|
        ff.semantic_errors *ff.object.errors.keys
        ff.input :children_name, label: "children name"
        ff.input :region_id, as: :nested_select,
              level_1: { attribute: :country_id },
              level_2: { attribute: :region_id }
              level_3: { attribute: :city_id }
      end
    end
  end
  f.actions
end
```

https://github.com/platanus/activeadmin_addons/issues/160
It is as same as above but I don't have  /app/assets/javascripts/activeadmin_addons/inputs/nested-select.js or  /app/assets/javascripts/activeadmin_addons folder. so how to manage this issue?

In which version of activeadmin_addons it is fixed?
Thanks.

**Comment by rjherrera** [comment]
@keyurvi I'm not sure if I understand your issue, can you explain a little bit more? Or maybe give some steps to reproduce this issue? I'd be happy to help.

**Comment by jgmontoya** [comment]
@keyurvi I have a feeling that your issue may have been solved by #196 

If that's not the case please explain it a bit further.

**Comment by jgmontoya** [comment]
I'm closing this issue, please open a new one if needed.
